src/atomic.py

```diff
@@ -132,6 +132,11 @@ def say_text(text_node):
 	evl.say_text(text)
 	return TruthValue(1, 1)
 
+def greet_recognized_face(recog_id_node):
+	text = "Hello " + recog_id_node.name
```

Comment:
Ugh. This is a complete short circuit of the language pipeline.  Its nasty, even as a short-term hack.